schema/nanotdf/README.md

```diff
@@ -385,13 +399,13 @@ object described in [Section 3.4.1] is used to describe the remote policy.
 
 ###### 3.4.2.3.2 Body for Embedded Policy
 
-These policy types allow for creation and binding of arbitraty policies. 
+These policy types allow for creation and binding of arbitrary policies.
 
-| Section                       | Minimum Length (B) | Maximum Length (B)  |
-|-------------------------------|--------------------|---------------------|
-| Content Length                | 2                  | 2                   |
-| Plaintext/Ciphertext          | 1                  | 255                 |
-| (Optional) Policy Key Access  | 36                 | 136                 |
+| Section                   | Minimum Length (B) | Maximum Length (B) |
+| ------------------------- | ------------------ | ------------------ |
+| Content Length            | 2                  | 2                  |
+| Plaintext/Ciphertext      | 1                  | 64,000             |
+| (Optional) Policy Binding | 8                  | 132                |
```

Comment:
Shouldn't this be policy key access as it was before? I think this is reference to enum 0x03, "Embedded Policy (Encrypted W/[Policy Key Access])

This basically allows one to encrypt their policy and have a key access scheme that differs from that of the primary object key.